tallycash/extension
# Rename polygon weth and eth pos

# What? 

Rename the current `ETH - POS` and `Weth` tokens from the polygon token list to match the user's expectations. 

# Test

- Test the fix on new install / fresh fetch
  - install wallet 
  - open devtools > Application > indexedDB > `tally/indexing` > tokenLists
  - check that in all the instances where the name is `Popular Tokens` list > 
    - ⚠️ There should be 2 of these in the db. The 2nd one arrives a bit after the install. To see it, you need to refresh the db view! ![CleanShot 2022-11-10 at 11 54 13](https://user-images.githubusercontent.com/7690112/201072779-69c2711d-cf50-44e4-a824-5c1f29662831.png)
    - tokens[0] ![CleanShot 2022-11-10 at 11 51 46](https://user-images.githubusercontent.com/7690112/201072210-171bdcf6-2835-408e-b9e0-7074273e9754.png)
    - tokens[28] ![CleanShot 2022-11-10 at 11 52 09](https://user-images.githubusercontent.com/7690112/201072287-73ee56a9-6580-4e7d-86d7-a5986a21830c.png)
- Test the migration
  - remove the extension 
  - comment out [these lines](https://github.com/tallycash/extension/blob/83cb6ad88d3b19797ccdeb822a33ade2b37ba98e/background/services/indexing/index.ts#L712-L716)
  - `yarn start` > install the extension
  - check the same thing as in the previous point 
- Check UI 
  -  on a fresh install 
  - import seed 
  - on swap page
  - check you have 
    - ![CleanShot 2022-11-10 at 11 58 01](https://user-images.githubusercontent.com/7690112/201073606-b3bb14dd-b447-414b-96a8-15aa0b01e004.png)
    - ![CleanShot 2022-11-10 at 11 58 20](https://user-images.githubusercontent.com/7690112/201073705-a1cf3b84-f16f-4e5b-879d-7028306ee238.png)
    - swap to `WETH` and check it on polygonscan
      - [example tx](https://polygonscan.com/tx/0x9db07e80182b737fb48c4c87c145b006c9c07eb37216771a0a19b1e1604174cf)

Closes #2583

Latest build: [extension-builds-2586](https://github.com/tallycash/extension/suites/9233860812/artifacts/431598179) (as of Thu, 10 Nov 2022 12:07:52 GMT).
